src/main/java/com/github/dockerjava/api/model/Node.java

```diff
@@ -8,20 +9,30 @@
  * A node as returned by the /events API, for instance, when Swarm is used.
  */
 @JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Node {
 
     @JsonProperty("Name")
     private String name;
 
-    @JsonProperty("Id")
+    @JsonProperty("ID")
```

Comment:
>It may happen that different calls has different internals in the same Node object, then it should be splitted. 
 
Docker api has no `Node` response, Node is only for *swarm*.  I am still trying to figure out what backward compatibility this breaking. 

Is it merely your conjecture at this point?